esmerald/applications.py

```diff
@@ -720,13 +720,18 @@ def is_valid(number: int) -> bool:
             ),
         ] = None,
         static_files_config: Annotated[
-            Optional["StaticFilesConfig"],
+            Union[
+                "StaticFilesConfig",
+                list["StaticFilesConfig"],
```

Comment:
I would use List and Tuple from typing here instead of the native since we do still support python 3.9?

Reply:
python 3.9 has support for this, or?

Reply:
checked this:
https://docs.python.org/3/whatsnew/3.9.html#type-hinting-generics-in-standard-collections